### PR TITLE
Add instruction cards to intro page

### PR DESCRIPTION
**Pull request by emilyzhaoo**
Add instruction cards to intro page.

Cards describe features of the website.
Add icon button for easy navigation of website.
Add CSS styling to intro page.

### CODE REVIEW

**Review by zhoua0805 — APPROVED** [review]
LGTM

**Review by EkkiD — COMMENTED** [review]
A screenshot might be helpful for these changes that are very visual

**Comment by emilyzhaoo** [comment]
Screenshot 2020-07-30 at 5 39 27 PM
Intro page styling.

image
Add instruction cards to intro page.

File: planet/src/main/webapp/index.html [hunk 1]
```diff
@@ -41,10 +50,45 @@ <h1>Get ready to
...
+                </div>            
+            </div>
+            <div class="column center">
+                <div class="card">
+                    <h3>Step 3:</h3>
+                    <h3>Generate an Itinerary</h3>
+                    <hr/>
+                    <p>On the Itinerary page, pick and choose from your saved places to generate an itinerary. Enter in an estimated visit time and create custom 
+                        events to add. Choose between optimized and ordered itineraries to generate.</p>
+                    <a href="itinerary.html" class="go waves-effect waves-light btn red accent-2">GO</a>
```
Comment (EkkiD) [inline planet/src/main/webapp/index.html]: What happens if the user does not have any places saved and they click through to here? Will they be ushered back to the places page?

File: planet/src/main/webapp/index.html [hunk 2]
```diff
@@ -41,10 +50,45 @@ <h1>Get ready to
...
+                </div>                
+            </div>
+            <div class="column center">
+                <div class="card">
+                    <h3>Step 2:</h3>
+                    <h3>Search Destinations</h3>
+                    <hr/>
+                    <p>Navigate to the Planet home page and start searching and saving destinations. View your searched locations on our map and browse through a list
+                        with details about each place.</p>
+                    <a href="places.html" class="go waves-effect waves-light btn red accent-2">GO</a>
```
Comment (EkkiD) [inline planet/src/main/webapp/index.html]: Consider adding different text to the different buttons. Maybe something like "Get Started", "Start Searching", and "Plan"?

Comment (EkkiD) [reply planet/src/main/webapp/index.html]: (Feel free to choose better wording)

**Review by EkkiD — CHANGES_REQUESTED** [review]

File: planet/src/main/webapp/index.html (hunk 1 above)
Comment (emilyzhaoo) [reply planet/src/main/webapp/index.html]: The users will still be able to see the itinerary page even when not signed it! I think I will work with Alice to display a message that prompts the user to sign in to actually be able to access saved places and create an itinerary

File: planet/src/main/webapp/index.html (hunk 2 above)
Comment (emilyzhaoo) [reply planet/src/main/webapp/index.html]: Sounds good! will do that

**Review by EkkiD — APPROVED** [review]
LGTM

**Pull request merged**